azext_edge/edge/vendor/clients/iotopsmgmt/_client.py

```diff
@@ -19,6 +19,9 @@
     BrokerAuthorizationOperations,
     BrokerListenerOperations,
     BrokerOperations,
+    DataFlowEndpointOperations,
```

Comment:
did you not generate everything the first time or is this a newer version of the swagger?

Reply:
This is a newer API spec. If you look at the PR, the API is incremented.